buildSrc/elastic.importorder

```diff
@@ -0,0 +1,8 @@
+#Organize Import Order
```

Comment:
I assume this is an eclipse generated file. Maybe it would make sense to mention this in this file somewhere to know what its used for?

Reply:
++